databases/de-database-schema/src/main/tables/76_job_status_updates.sql

```diff
@@ -0,0 +1,48 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records job status updates
+--
+CREATE TABLE job_status_updates (
+    id uuid NOT NULL DEFAULT uuid_generate_v4(),
+
+    -- corresponds to the external_id field in the job_steps table
+    external_id character varying(64) NOT NULL,
```

Comment:
Hm. It does look like the code that joins on this in apps is unused (vestigial bits of the periodic-sync code). However, job-status-to-apps-adapter does do a select `WHERE external_id = $1` which without an index will need to look at every row of the job_status_updates table; this is in code which happens every time it tries to propagate anything up (as you might expect). To start this will not likely be an issue since the table will start out fairly small, but I think it would be wise to add an index on this column now anyway. "Number of job updates since the new jex was shipped" shouldn't be a linear performance problem.